### ISSUE TITLE
Dugnad trial at a collaboration café

### ISSUE THREAD

**Issue by JimMadge** [the-turing-way/the-turing-way#4013]
### Summary

I was thinking about this challenge,

- Our community is super enthusiastic about adding new pages and chapters
- Maintaining existing pages is a bit boring
- However, maintaining the quality of the book is really important

A member of my team has organised some [dugnad](https://thenorwayguide.com/norwegian-dugnad-explained/) events, where as a group we work to tidy and fix things.
Maybe this is something we could try as a community?

It could work a bit like the alt text sessions, as a way to get people making straightforward contributions which have a big impact on the quality of the book.
Also, some sort of gamification, or building a social event around this could help encourage experienced contributors to get involved in maintenance.

Everyone gets a file and works through a checklist of tasks like,

- Check spelling and grammar
- Simplify complex or confusing sentences
- Link terms to glossary, or add terms to the glossary
- Add alt text to images
- Open issues to flag larger tasks like, out of date content, contradictory content, and so on

If you finish your file, you can move on to another or join someone else.

### What needs to be done?

- [ ] Organise the infrastructure
  - [ ] Instructions
  - [ ] Checklist of tasks
  - [ ] Timing for event
  - [ ] Mentor/support role outlines
- [ ] Decide a collaboration café to trial a dugnad
- [ ] Promote the event

### Who can help?

* @aleesteele
* @EKaroune (insights from running alt test sessions would be really helpful)

---

**Comment by aleesteele** [comment]
@JimMadge - how about we trial this at an upcoming Collaboration Cafe?

**Comment by JimMadge** [comment]
Yes that would be great. If we can get some experienced contributors to join that would be good for feedback.

One problem I can foresee is assuming too much familiarity and not providing enough support for people to contribute.

**Comment by Arielle-Bennett** [comment]
Noting that we discussed this idea in the Collab Cafe on 2 April 2025 and agreed it would be good to plan an initiative for the Nov 2025 Book Dash around this!
